### PR TITLE
feat(mikro-orm): Add EntityManager decorator

### PR DESCRIPTION
**Pull request by Romakita**
Hello @derevnjuk 

This PR add `@Em` and `@EntityManager`.

I also opened a PR on Mikro ORM to add Ts.ED integration page on his website: https://github.com/mikro-orm/mikro-orm/pull/2587

See you :)
Romain

### CODE REVIEW

File: packages/orm/mikro-orm/test/helpers/services/UserService.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import {Injectable} from "@tsed/di";
 import {Connection} from "../../../src";
-import {MikroORM} from "@mikro-orm/core";
+import {EntityManager, MikroORM} from "@mikro-orm/core";
+import {Em} from "../../../src/decorators/entityManager";
```
Comment (derevnjuk) [inline packages/orm/mikro-orm/test/helpers/services/UserService.ts]: Use shorthand imports here. Also, it lets us optimize imports merging 2th and 4th lines 

```suggestion
import {Em} from "../../../src";
```

File: packages/orm/mikro-orm/src/decorators/entityManager.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import {MikroOrmRegistry} from "../services";
+
+/**
+ * Get the entity manager for the given connection name.
+ * @param connectionName
+ * @decorator
+ * @mikroOrm
+ */
+export const EntityManager = (connectionName?: string): PropertyDecorator =>
+  Inject(MikroOrmRegistry, (registry: MikroOrmRegistry) => registry.get(connectionName).em) as PropertyDecorator;
```
Comment (derevnjuk) [inline packages/orm/mikro-orm/src/decorators/entityManager.ts]:
```suggestion
  Inject(MikroOrmRegistry, (registry: MikroOrmRegistry) => registry.get(connectionName)?.em) as PropertyDecorator;
```

File: packages/orm/mikro-orm/readme.md [hunk 3]
```diff
@@ -112,6 +129,67 @@ export class UsersService {
...
+  async create(user: User): Promise<User> {
+    await this.em.persistAndFlush(user);
+    console.log("Saved a new user with id: " + user.id);
+
+    return user;
+  }
+}
+```
+
+> Notice that the `EntityManager` is imported from the `@mikro-orm/driver` package, where driver is `mysql`, `sqlite`, `postgres` or what driver you are using.
```
Comment (derevnjuk) [inline packages/orm/mikro-orm/readme.md]: I don't think we have to force a user to specify a concrete type, he is able to declare a base type as well (i.e. `EntityManager` from `@mikro-orm/core`)

**Review by derevnjuk — CHANGES_REQUESTED** [review]

**Comment by derevnjuk** [comment]
@Romakita excellent, it is a good idea 💪🏾  I propose you convert mikro-orm/mikro-orm#2587 to draft until #1697 is merged.

File: packages/orm/mikro-orm/readme.md (hunk 3 above)
Comment (Romakita) [reply packages/orm/mikro-orm/readme.md]: This note come from the official website :) I don't really understand the final goal of this ^^. I can remove it.

Comment (Romakita) [reply packages/orm/mikro-orm/readme.md]: done

File: packages/orm/mikro-orm/src/decorators/entityManager.ts (hunk 2 above)
Comment (Romakita) [reply packages/orm/mikro-orm/src/decorators/entityManager.ts]: done

File: packages/orm/mikro-orm/test/helpers/services/UserService.ts (hunk 1 above)
Comment (Romakita) [reply packages/orm/mikro-orm/test/helpers/services/UserService.ts]: done

**Comment by Romakita** [comment]
@derevnjuk I fixed the production branch. yesterday I started my documentation branch from your repository by error ^^.

To fix your issue on #1698 and #1697, simply fetch the production branch from my repo and rebase your branch from stream/production (stream => tsedio).

See you

**Review by derevnjuk — APPROVED** [review]
LGTM 💪🏾

**Pull request merged**